index.html

```diff
@@ -6,6 +6,10 @@
 
   <body>
     <div id="heads-area">
+      <div class="person-frame-inner">
+        <img class="ball" src="desktop/OSU/mia.jpg" />
```

Comment:
You need to add the path of the image that is now in our GitHub repository, because we don't have access to the desktop of your personal computer. Hint: `images/mia.jpg`.